README.md

```diff
@@ -100,11 +100,11 @@ A sample configuration is provided as [config.dist.yml](config.dist.yml).
 
 | Name                   | Description                                                                                                                   | Default | Required? |
 | ---------------------- | ----------------------------------------------------------------------------------------------------------------------------- | ------- | --------- |
-| `API_KEY_HASH`         | Argon2id hash of the API key to be used. <br> Can be generated [here](https://argon2.online/). Make sure to use Encoded Form. | -       | yes       |
+| `API_KEY_HASHES`         | Argon2id hash of the API key to be used. <br> Can be generated [here](https://argon2.online/). Make sure to use Encoded Form. | -       | yes       |
 | `CORS_ALLOWED_ORIGINS` | List of allowed CORS origins                                                                                                  | -       | yes       |
 | `CORS_ALLOWED_METHODS` | List of allowed CORS methods                                                                                                  | `GET`   | no        |
 
 ### Overriding options
 
 Configuration options from the configuration file can be overwritten via environment variables.  
-Note that overriding only works for config options which have a single value (e.g. `API_KEY_HASH`) and not options that are a list (e.g. `CORS_ALLOWED_ORIGINS`)
+Note that overriding list values (arrays), works using ';' as the separator. For example, `API_KEY_HASHES='HASH_1;HASH:2'`.
```

Comment:
Muss man bisschen vorsichtig sein mit der Formulierung, weil z.B. bei `CORS_ALLOWED_ORIGINS` funktioniert das nicht, weil quasi jedes Zeichen Teil einer URL sein kann.
Glaube es wäre deswegen sinnvoll hier noch nen entsprechenden Hinweis einzubauen, dass das bei `CORS_ALLOWED_ORIGINS` nicht geht

Reply:
> A [semicolon is reserved](http://www.ietf.org/rfc/rfc1738.txt) and should only for its special purpose (which depends on the scheme).

Halte `;` in einer URL für sehr unwahrscheinlich:
> A [semicolon is reserved](http://www.ietf.org/rfc/rfc1738.txt) and should only for its special purpose (which depends on the scheme).

Ist natürlich keine feste Garantie, aber glaube eher weniger, dass da jemand in einen Fehler läuft

Reply:
Überzeugt. Man kanns ja immernoch ändern wenn es zu einem Problem wird